config/routes.rb

```diff
@@ -434,6 +434,10 @@ def healthcare_routes
     end
   end
 
+  # Client search queries
+  resources :client_search_queries, only: [:create], controller: 'clients/search_queries'
+  get '/clients/:id/search', to: 'client_access_control/clients#search', as: 'client_search_query'
```

Comment:
Are we not worried this might conflict with other client pages (eventually?) the `id` in this isn't the same as the id in `/clients/:id`.

Reply:
It's a good point. I think the chance of a routing conflict is low unless there's another search action on client record. But I think we can do better. I'll put these both under 'client_searches'
```
  resources :client_searches, only: [:create], controller: 'clients/search_queries'
  get '/client_searches/:id', to: 'client_access_control/clients#search', as: 'client_search_query'
```